### PR TITLE
Add ESV, LEB, and YLT NTs aligned to SBLGNT

### PR DESCRIPTION
**Pull request by RickBrannan**

### CODE REVIEW

File: data/alignments/eng/LEB/SBLGNT-LEB-manual.toml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+name.eng = "Lexham English Bible"
+url = "http://www.lexhamenglishbible.com"
+
+[target.metadata]
+copyright = "Copyright (c) 2023 by Clear Bible, Inc."
+license = "CC-BY 4.0"
+
+[alignment]
+
+identifier = "NA27-LEB-manual"
```
Comment (sboisen) [inline data/alignments/eng/LEB/SBLGNT-LEB-manual.toml]: @RickBrannan  By convention, this `identifier` should match the alignment filename, in this case `SBLGNT-LEB-manual`.

Comment (RickBrannan) [reply data/alignments/eng/LEB/SBLGNT-LEB-manual.toml]: Hm, I thought I changed that with the second commit. Will look again.

File: data/alignments/eng/LEB/SBLGNT-LEB-manual.toml [hunk 2]
```diff
@@ -23,7 +23,7 @@ license = "CC-BY 4.0"
 
 [alignment]
```
Comment (RickBrannan) [inline data/alignments/eng/LEB/SBLGNT-LEB-manual.toml:25]: @sboisen , this is the second commit where I fixed the issue.

File: data/alignments/eng/ESV/SBLGNT-ESV-manual.toml [hunk 3]
```diff
@@ -21,7 +21,7 @@ license = "CC-BY 4.0"
 
 [alignment]
 
-identifier = "NA27-ESV-manual"
+identifier = "SBLGNT-ESV-manual"
 format = "grapecity"
 license = "CC-BY-4.0"
 process = "manual"
```
Comment (RickBrannan) [inline data/alignments/eng/ESV/SBLGNT-ESV-manual.toml:27]: @sboisen this is the second commit where I addressed that problem (after I noticed it, totally missed it the first time).

**Comment by RickBrannan** [comment]
So @sboisen had some initial feedback, which was addressed, so I'm assuming this is all OK and merging the PR.

**Pull request merged**